libwebrtc/src/native/audio_source.rs

```diff
@@ -71,32 +71,42 @@ impl NativeAudioSource {
                 let mut interval = interval(Duration::from_millis(10));
                 interval.set_missed_tick_behavior(livekit_runtime::MissedTickBehavior::Delay);
                 let blank_data = vec![0; samples_10ms];
+                let send_silent_frames = !source.sys_handle.audio_options().disable_silent_frames;
 
                 loop {
-                    interval.tick().await;
```

Comment:
I think we should keep the interval when send_silent_frames is enabled?

Should we rename this feature: enable_queue instead of disable_silent_frames? Thinking about how users will understand it

Reply:
I moved the interval here:
https://github.com/livekit/rust-sdks/pull/353/files#diff-8ab7d3dca0e75d71ebc9798430e49ba444958ba064fb95ff1e00e7fde1696611R99

Makes sense. If we change the name to `enable_queue`, in the case of false, should we directly skip sending the `frame` to `po_tx`when the audio frame enters `capture_frame`? This way, we can actually skip the `queue` and send the frame to `libwebrtc`.

Reply:
oh sorry I missed this line.
Yeah we can directly skip the queue, and send to libwebrtc